doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -71,11 +71,11 @@ can be used to force the detect engine to tie a transaction
 to an alert.
 This transaction is not guaranteed to be the relevant one,
 depending on your use case and how you define relevant here.
-If there are multiple live transactions, none will get
-picked up.
-The alert event will have ``"tx_guessed": true`` to recognize
-these alerts.
-
+**WARNING: If there are multiple live transactions, none will get
+picked up.** This is to reduce the chances of logging unrelated data, and may
+lead to alerts being logged without metadata, in some cases.
+The alert event will have ``tx_guessed: true`` to recognize
+such alerts.
```

Comment:
Do you want to add there that we also log some transaction metadata for rule with stream matches ?
And that the first transaction is logged, like one TCP packet with 3 DNS requests to suricata.io  oisf.net and suricon.net and rule `alert tcp any any -> any any (content: "suricon"; sid: 1)` will log the request to suricata.io

Reply:
Seems all too much of a low level implementation detail to discuss in configuration docs.  But these types of issues are making me think we need a know issues and shortcomings chapter.